common/djangoapps/student/admin.py

```diff
@@ -511,6 +515,25 @@ class Meta(object):
         model = AllowedAuthUser
 
 
+@admin.register(CourseEnrollmentCelebration)
+class CourseEnrollmentCelebrationAdmin(DisableEnrollmentAdminMixin, admin.ModelAdmin):
+    """Admin interface for the CourseEnrollmentCelebration model. """
+    raw_id_fields = ('enrollment',)
+    list_display = ('id', 'course', 'user', 'celebrate_first_section')
+    search_fields = ('enrollment__course__id', 'enrollment__user__username')
+
+    class Meta(object):
+        model = CourseEnrollmentCelebration
+
+    def course(self, obj):
+        return obj.enrollment.course.id
+    course.short_description = 'Course'
```

Comment:
Right above with the `def course`. When python reads this class, it does so in order and defines the `course` attribute, then the line right after that modifies the `course` attribute's description.

Reply:
Don't you need to specify this as an `@property`? Because course is a function. I feel like this is making me lose my mind a little MT. I would potentially understand `course().short_description`...

Reply:
Ah yeah, it is a function. But paste this into a local python terminal and you'll see that functions are just like any other python object: you can set properties on 'em:

```
def blarg(): 
    pass

blarg
blarg.foo = 'hi'
blarg.foo
```

I guess a `short_description` property is how Django knows what the header for a custom admin function is supposed to be? I copied the flow from elsewhere in this file I think.

Reply:
That is absolutely wild.